### :sob: 테스트가 실패했습니다.
[GitHub Actions 확인하기](https://github.com/depromeet/toks-web/actions/runs/4404112859)


- 패키지명: @depromeet/layout
- 위치: shared/layout


stdout:
```

/home/runner/work/toks-web/toks-web/shared/layout/src/components/UserMenu.tsx
  2:38  error  Member 'IMAGE_URL' of the import declaration should be sorted alphabetically  sort-imports

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.

```
에러 트레이스 (stderr):
```
Command failed with exit code 1: yarn workspace @depromeet/layout run lint
```

--- a/shared/layout/src/components/UserMenu.tsx
+++ b/shared/layout/src/components/UserMenu.tsx
@@ -1,5 +1,5 @@
 import { theme } from '@depromeet/theme';
-import { BP, Image, MAX_WIDTH, Text, IMAGE_URL } from '@depromeet/toks-components';
+import { BP, IMAGE_URL, Image, MAX_WIDTH, Text } from '@depromeet/toks-components';
 import styled from '@emotion/styled';
 import { Flex, Spacing } from '@toss/emotion-utils';
 

